Lib-DNDBeyond/dndb_CastSpell_processor.command

```diff
@@ -0,0 +1,51 @@
+[h: log.debug (getMacroName() + ": args = " + json.indent (macro.args))]
+[h: actionButton = json.get (macro.args, "actionButton")]
+[h: spellSlot = json.get (macro.args, "spell-slot")]
+[h, if (spellSlot == ""): spellSlot = 0; ""]
```

Comment:
TODO: someday create a Library of utilities that includes a 'getNumber', which coerces "" to 0. This pattern is all over the place.